### PR TITLE
feat(crons): Add ability to mute environments

### PR DESCRIPTION
**Pull request by davidenwang**
https://github.com/getsentry/sentry/assets/9372512/119719c5-9662-4f26-b841-7d0a254852f2

exists only on the timeline page for now, hopefully can get both delete/mute into the details page soon after

### CODE REVIEW

**Review by gaprl — APPROVED** [review]

File: static/app/actionCreators/monitors.tsx [hunk 1]
```diff
@@ -66,3 +66,29 @@ export async function updateMonitor(
...
+  monitorSlug: string,
+  environment: string,
+  isMuted: boolean
+) {
+  addLoadingMessage();
+
+  try {
+    const resp = await api.requestPromise(
+      `/organizations/${orgId}/monitors/${monitorSlug}/environments/${environment}`,
+      {method: 'PUT', data: {isMuted}}
```
Comment (rjo100) [inline static/app/actionCreators/monitors.tsx:82]: does this send a `key/value` of `isMuted: true/false`?

Comment (davidenwang) [reply static/app/actionCreators/monitors.tsx:82]: yup!

**Comment by gaprl** [comment]
Small detail we can change later -- let's keep "delete environment" at the bottom of the list, so it's like this:

1. View environment
2. Mute environment
3. Delete environment

Would be nice for us to add an icon later as well.

**Comment by davidenwang** [comment]
> Small detail we can change later -- let's keep "delete environment" at the bottom of the list, so it's like this:
> 
> 1. View environment
> 2. Mute environment
> 3. Delete environment
> 
> Would be nice for us to add an icon later as well.

oh lemme change the order now!

**Pull request merged**